src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -20,23 +25,54 @@
   }
 }
 
-:host(.highlight[selected]) {
+.highlight[selected] {
   border-left-color: blue;
   &:host(.calcite--rtl) {
     border-left-color: none;
     border-right-color: blue;
   }
 }
 
+.icon {
+  color: var(--calcite-app-foreground-link);
+  flex: 0 0;
+  line-height: 0;
+  width: var(--calcite-app-icon-size);
+  margin: 0 var(--calcite-app-side-spacing-half);
+}
+
 .label {
-  flex-grow: 1;
-  margin: 0 12px;
-  min-width: 60px;
+  display: flex;
+  flex-flow: column nowrap;
+  flex: 1 0;
+  overflow: hidden;
+  padding: 0 var(--calcite-app-side-spacing-half);
+  pointer-events: none;
+}
+
+.title {
+  // ------------------------
+  // ellipse after two lines
```

Comment:
Could this be reused? If so, a (future) mixin could help with reuse and maintainability. It may be hard to distinguish which props/values are for the ellipsis and which ones aren't when updates are made to the declaration.